src/mantine-core/src/components/Alert/Alert.tsx

```diff
@@ -29,6 +29,9 @@ export interface AlertProps
 
   /** Close button aria-label */
   closeButtonLabel?: string;
+
+  /** Alert Radius */
```

Comment:
Change description to: `Radius from theme.radius, or number to set border-radius in px`